src/test/java/gov/usds/case_issues/controllers/CaseDetailsApiControllerTest.java

```diff
@@ -278,7 +300,13 @@ public void addNoteToSnooze_activeCase_badRequest() throws Exception {
 
 	private TroubleCase initSampleCase() {
 		CaseType type = _dataService.ensureCaseTypeInitialized("T2", "Ahnold", "Metal and scary");
-		return _dataService.initCase(_sys, SAMPLE_CASE, type, ZonedDateTime.now());
+		TroubleCase troubleCase = _dataService.initCase(_sys, SAMPLE_CASE, type, ZonedDateTime.now());
+
+		String createdBy = troubleCase.getCreatedBy();
+		UserInformation user = new UserInformation(createdBy, "Admin Anna");
```

Comment:
Nitpick of the nittest: since this isn't (I don't think?) driven by the demo-user login code, probably a different name would be safer, to avoid confusion?